I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script_b_WeekLoop.rpy", line 39, in script
    if (HON_progress + act_cnt[2]) > HON_max[week_num]:
  File "game/script_b_WeekLoop.rpy", line 39, in <module>
    if (HON_progress + act_cnt[2]) > HON_max[week_num]:
TypeError: 'int' object has no attribute '__getitem__'

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/script_b_WeekLoop.rpy", line 39, in script
    if (HON_progress + act_cnt[2]) > HON_max[week_num]:
  File "C:\Users\Benjamin Shu\Documents\renpy-6.99.14.1-sdk\renpy\ast.py", line 1762, in execute
    if renpy.python.py_eval(condition):
  File "C:\Users\Benjamin Shu\Documents\renpy-6.99.14.1-sdk\renpy\python.py", line 1944, in py_eval
    return py_eval_bytecode(code, globals, locals)
  File "C:\Users\Benjamin Shu\Documents\renpy-6.99.14.1-sdk\renpy\python.py", line 1937, in py_eval_bytecode
    return eval(bytecode, globals, locals)
  File "game/script_b_WeekLoop.rpy", line 39, in <module>
    if (HON_progress + act_cnt[2]) > HON_max[week_num]:
TypeError: 'int' object has no attribute '__getitem__'

Windows-8-6.2.9200
Ren'Py 7.1.3.1092
SBU Senior Project 1.0
Wed Feb 13 01:39:06 2019
